CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
-# Change Log
+# Changelog
 
+All notable changes to this template will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## v0.2 2021-10-?
+- Updated the template based on improvements made to the repo during the SnowEx 2021 Hackweek
+
+### Changed
+- Automated build actions
+### Removed
+
+### Added
 ## v0.1 2021-?-?
```

Comment:
I'm going to modify the CHANGELOG.md on the main branch and a tag so we know what was used for snowex, you'll have to resolve the merge conflict here

Reply:
Merge conflict resolved - now we just need to fill in the info for this PR.

Reply:
Just kidding - I'm not sure what happened, but this is definitely not what the API text showed when I told GH to commit!